### PR TITLE
[line] growth vs factor option for 'Period Ratio'

### PR DESCRIPTION
**Pull request by mistercrunch**
screen shot 2016-08-17 at 8 58 13 am

### CODE REVIEW

File: caravel/forms.py [hunk 1]
```diff
@@ -817,6 +817,14 @@ def __init__(self, viz):
                     "[integer] Number of period to compare against, "
                     "this is relative to the granularity selected")
             }),
+            'period_ratio_type': (SelectField, {
+                "label": _("Period Ratio Type"),
+                "default": 'growth',
+                "choices": self.choicify(('factor', 'growth')),
```
Comment (xrmx) [inline caravel/forms.py]: i18n?

**Comment by mistercrunch** [comment]
@xrmx good catch, I made the options translatable

**Comment by ascott** [comment]
💯

File: caravel/viz.py [hunk 2]
```diff
@@ -1038,7 +1038,14 @@ def get_df(self, query_obj=None):
         num_period_compare = form_data.get("num_period_compare")
         if num_period_compare:
             num_period_compare = int(num_period_compare)
-            df = (df / df.shift(num_period_compare)) - 1
+            prt = form_data.get('period_ratio_type')
```
Comment (bkyryliuk) [inline caravel/viz.py]: @mistercrunch 
would it make sense to add tests to the viz.py functions?

Comment (xrmx) [inline caravel/viz.py]: It would be awesome to know that viz are rendering :)

**Pull request merged**